formValidation.js

```diff
@@ -0,0 +1,21 @@
+function createNode(type, nodeClass) {
+  const node = document.createElement(type);
+  if (nodeClass) node.className = nodeClass;
+  return node;
+}
+
+const formStatus = document.getElementById('fs-frm');
+const formEmail = document.getElementById('email-address');
+const formButton = document.getElementById('form-submit-button');
+const alertLowercase = formButton.appendChild(createNode('label', 'email-error'));
```

Comment:
- [ ] Good job implementing the error message but now another issue has appeared. Well, when the user clicks on the submit button will append a new error message and the error message will be shown several times kindly fix that in order to make the error message appear just one time.
 ![Captura de pantalla_2021-10-15_20-10-10](https://user-images.githubusercontent.com/53111464/137568119-9c1b88d9-2160-4350-8388-528b1ba304cb.png)